xchange-stream-kraken/src/main/java/info/bitrich/xchangestream/kraken/KrakenStreamingTradeService.java

```diff
@@ -198,7 +198,7 @@ else if ("stop".equals(orderType))
                 .type(KrakenAdapters.adaptOrderType(KrakenType.fromString(dto.type)))
                 .price(dto.price)
                 .feeAmount(dto.fee)
-                .feeCurrency(currencyPair.base)
+                .feeCurrency(currencyPair.counter)
```

Comment:
isn't it actually dependant on the orders type?

Reply:
The documentation of class KrakenOwnTrade states "_fee - total fee (quote currency)_".
And the documentation in [Kraken's Websocket API](https://docs.kraken.com/websockets/#message-ownTrades) also states "_fee - decimal - total fee (quote currency)_".

Sounds like it always relates to the qoute/counter currency of the currency pair and not to the order type.